dv/uvm/core_ibex/directed_tests/riscv_test.h

```diff
@@ -0,0 +1,274 @@
+// See LICENSE for license details.
```

Comment:
As with `encoding.h` I think this has come from some other RISCV repo? The same comments apply in that this should really be vendored in rather than included directly here.

Reply:
Added a patch file for it since some modifications was required for test start sequence of instructions, test ending mechanism for Ibex.